src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -948,6 +953,14 @@ protected function build_attendee_data( array $attendee, $context = 'default' )
 			'modified'          => $attendee_post->post_modified,
 			'modified_utc'      => $attendee_post->post_modified_gmt,
 			'rest_url'          => $main->get_url( '/attendees/' . $attendee_id ),
+			'ticket' => [
+				'id'          => $ticket->ID,
+				'title'       => $ticket->name,
+				'description' => $ticket->description,
+				'price'       => $ticket->price,
```

Comment:
Given that we may need to show the formatted price, having the formatted number would be a good idea (maybe we can keep a reserved place for the raw number).

Reply:
Updated 👍